tutorials/tutorial1.md

```diff
@@ -153,10 +158,10 @@ To train the model, we supply the training and validation datasets as well as th
 python $atacworks/scripts/main.py train \
         --config configs/train_config.yaml \
         --config_mparams configs/model_structure.yaml \
```

Comment:
let's use absolute path: `$atacworks/tutorial1/configs/model_structure.yaml`

Reply:
This would mean that the tutorial has to be run in the AtacWorks folder. I'm not sure this is a good idea, users should be able to download the tutorial files anywhere.

Reply:
Okay makes sense. I'll then figure this out in the next PR. Let's leave this as is.